tests/upgrades/test_remoteexecution.py

```diff
@@ -0,0 +1,165 @@
+"""Test for Capsule related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import os
+from wait_for import wait_for
+
+from nailgun import entities
+from robottelo.constants import DEFAULT_LOC_ID, DISTRO_RHEL7
+from robottelo.helpers import add_remote_execution_ssh_key
+from robottelo.vm import VirtualMachine
+from robottelo.test import APITestCase
+from upgrade_tests import post_upgrade, pre_upgrade
+from upgrade_tests.helpers.scenarios import create_dict, get_entity_data
+
+
+class Scenario_remoteexecution_external_capsule(APITestCase):
+    """The test class contains pre-upgrade and post-upgrade scenarios to test if
+        remote execution working with external capsule.
+
+        Test Steps:
+
+        1. Before Satellite upgrade, Create VM,Subnet,install katello-ca,
```

Comment:
Devide these steps on multiple lines instead of putting thin in one line.
Also,
s/VM/content host/